monkey/infection_monkey/telemetry/file_encryption_telem.py

```diff
@@ -1,13 +1,11 @@
-from pathlib import Path
-
 from common.common_consts.telem_categories import TelemCategoryEnum
 from infection_monkey.telemetry.base_telem import BaseTelem
 from infection_monkey.telemetry.batchable_telem_mixin import BatchableTelemMixin
 from infection_monkey.telemetry.i_batchable_telem import IBatchableTelem
 
 
 class FileEncryptionTelem(BatchableTelemMixin, IBatchableTelem, BaseTelem):
-    def __init__(self, filepath: Path, success: bool, error: str):
+    def __init__(self, filepath: str, success: bool, error: str):
```

Comment:
Why this change? All of the added calls to `os.path.expandvars()` are wrapped in `Path()`